packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -548,7 +556,12 @@ const CFDPasswordForm = ({
                                         i18n_default_text='Enter your {{platform}} password to add a {{platform_name}} {{account}} account.'
                                         values={{
                                             platform: getCFDPlatformLabel(platform),
-                                            platform_name: platform === CFD_PLATFORMS.MT5 ? 'MT5' : 'Deriv X',
+                                            platform_name:
+                                                platform === CFD_PLATFORMS.MT5
+                                                    ? 'MT5'
+                                                    : platform === CFD_PLATFORMS.DXTRADE
+                                                    ? 'Deriv X'
+                                                    : 'cTrader',
```

Comment:
same goes there use the same variable created upwards.
If would work fine and more viewable.